v2/devices.go

```diff
@@ -17,7 +25,7 @@ type (
 
 // SetDeviceSubnetRoutes sets which subnet routes are enabled to be routed by a device by replacing the existing list
 // of subnet routes with the supplied routes. Routes can be enabled without a device advertising them (e.g. for preauth).
-func (c *Client) SetDeviceSubnetRoutes(ctx context.Context, deviceID string, routes []string) error {
+func (c *Devices) SetDeviceSubnetRoutes(ctx context.Context, deviceID string, routes []string) error {
```

Comment:
Thoughts on making this `func (d *Devices)...`

And similar for all the others

Reply:
It had occurred to me too. Probably makes sense to rename.

Reply:
Done. The names might have to change if we don't go with the `Resource` suffix.